librz/core/cmd_print.c

```diff
@@ -4163,16 +4163,21 @@ static void _pointer_table(RzCore *core, ut64 origin, ut64 offset, const ut8 *bu
 			rz_cons_printf("af case.%d.0x%" PFMT64x " 0x%08" PFMT64x "\n", n, offset, addr);
 			rz_cons_printf("ax 0x%" PFMT64x " 0x%08" PFMT64x "\n", offset, addr);
 			rz_cons_printf("ax 0x%" PFMT64x " 0x%08" PFMT64x "\n", addr, offset); // wrong, but useful because forward xrefs dont work :?
+			// FIXME: "aho" doesn't accept anything here after the "case" word
 			rz_cons_printf("aho case 0x%" PFMT64x " 0x%08" PFMT64x " @ 0x%08" PFMT64x "\n", (ut64)i, addr, offset + i); // wrong, but useful because forward xrefs dont work :?
 			rz_cons_printf("ahs %d @ 0x%08" PFMT64x "\n", step, offset + i);
 		} else if (mode == '.') {
-			rz_core_cmdf(core, "af case.%d.0x%" PFMT64x " @ 0x%08" PFMT64x "\n", n, offset, addr);
-			rz_core_cmdf(core, "ax 0x%" PFMT64x " 0x%08" PFMT64x "\n", offset, addr);
-			rz_core_cmdf(core, "ax 0x%" PFMT64x " 0x%08" PFMT64x "\n", addr, offset); // wrong, but useful because forward xrefs dont work :?
-			// rz_core_cmdf (core, "CC+ case %d: 0x%08"PFMT64x " @ 0x%08"PFMT64x "\n", i / step, addr, origin);
-			rz_core_cmdf(core, "CCu case %d: @ 0x%08" PFMT64x "\n", n, addr); //, origin);
-			rz_core_cmdf(core, "aho case %d 0x%08" PFMT64x " @ 0x%08" PFMT64x "\n", n, addr, offset + i); // wrong, but useful because forward xrefs dont work :?
-			rz_core_cmdf(core, "ahs %d @ 0x%08" PFMT64x "\n", step, offset + i);
+			const char *case_name = rz_str_newf("case.%d.0x%" PFMT64x, n, offset);
+			rz_core_analysis_function_add(core, case_name, addr, false);
+			rz_analysis_xrefs_set(core->analysis, addr, offset, RZ_ANALYSIS_REF_TYPE_NULL);
+			rz_analysis_xrefs_set(core->analysis, offset, addr, RZ_ANALYSIS_REF_TYPE_NULL); // wrong, but useful because forward xrefs dont work :?
+			const char *case_comment = rz_str_newf("case %d:", n);
+			const char *comment = rz_meta_get_string(core->analysis, RZ_META_TYPE_COMMENT, addr);
+			if (!comment || (comment && !strstr(comment, case_comment))) {
```

Comment:
introduce a new API for `CCu` and re-use it here. This thing is otherwise error-prone and hard to re-do every time we have to replace `CCu`.

Reply:
Done. Note though, I didn't create an API for the whole `CCu` since it also has `base64:` processing. I think that part of argument processing probably belong to the newshell parser so that any argument of many different commands can be represented as base64-encoded string.